src/mlpack/bindings/julia/test_julia_binding_main.cpp

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file julia_binding_test.cpp
```

Comment:
Hey @rcurtin do we require this file.?

Reply:
I'm not sure what you mean; can you explain further?

Reply:
I think so here we have two main.cpp files.

Reply:
One Julia folder and one in tests folder.

Reply:
Oh!  Nice catch, thanks.  I removed the duplicate in 80c9a8d.